src/Robo/Tasks/RunTestsTask.php

```diff
@@ -0,0 +1,224 @@
+  }
+
+  /**
+   * @return $this
+   */
+  public function all() {
+    $this->option("all");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function browser() {
+    $this->option("browser");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function clean() {
+    $this->option("clean");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function color() {
+    $this->option("color");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function dieOnFail() {
+    $this->option("die-on-fail");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function keepResultsTable() {
+    $this->option("keep-results-table");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function keepResults() {
+    $this->option("keep-results");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function listFilesJson() {
+    $this->option("list-files-json");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function nonHtml() {
+    $this->option("non-html");
+    return $this;
+  }
+
+  /**
+   * @return $this
+   */
+  public function suppressDeprecations() {
+    $this->option("suppress-deprecations");
+    return $this;
+  }
+
+  /**
+   * @param int $concurrency
+   *
+   * @return $this
+   */
+  public function concurrency($concurrency) {
+    $this->option("concurrency", $concurrency);
+    return $this;
+  }
+
+  /**
+   * @param int $repeat
+   *
+   * @return $this
+   */
+  public function repeat($repeat) {
+    $this->option("repeat", $repeat);
+    return $this;
+  }
+
+  /**
+   * @param string $dbUrl
+   *
+   * @return $this
+   */
+  public function dbUrl($dbUrl) {
+    $this->option("dburl", $dbUrl);
+    return $this;
+  }
+
+  /**
+   * @param string $sqlite
+   *
+   * @return $this
+   */
+  public function sqlite($sqlite) {
+    $this->option("sqlite", $sqlite);
+    return $this;
+  }
+
+  /**
+   * @param string $url
+   *
+   * @return $this
+   */
+  public function url($url) {
+    $this->option("url", $url);
+    return $this;
+  }
+
+  /**
+   * @param string $xml
+   *
+   * @return $this
+   */
+  public function xml($xml) {
+    $this->option("xml", $xml);
+    return $this;
+  }
+
+  /**
+   * @param string $types
+   *
+   * @return $this
+   */
+  public function types($types) {
+    $this->option('types', $types, ' ');
+    return $this;
+  }
+
+  /**
+   * @param array $tests
+   *
+   * @return $this
+   */
+  public function tests($tests) {
+    $this->args($tests);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCommand() {
+    return $this->command . ' ' . $this->runTestsScriptCommand . $this->arguments;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function run() {
+    $this->printTaskInfo('Running run-tests.sh {arguments}', ['arguments' => $this->arguments]);
+    return $this->executeCommand($this->getCommand());
```

Comment:
This looks to be outputting the correct command, but something deeper in the call stack seems to be off.  Again, might need some eyes from someone more familiar with Robo than I am.

Reply:
```
[Acquia\Blt\Robo\Tasks\RunTestsTask] Running run-tests.sh  --xml /var/www/html/reports/phpunit --verbose --all --url 'http://local.html.com' --types PHPUnit-Unit User System
[Acquia\Blt\Robo\Tasks\RunTestsTask] Running /usr/bin/php ./core/scripts/run-tests.sh --xml /var/www/html/reports/phpunit --verbose --all --url 'http://local.html.com' --types PHPUnit-Unit User System in /var/www/html/docroot

Drupal test run
---------------

All tests will run.

Test run started:
  Friday, November 16, 2018 - 22:40

Test summary
------------


In Application.php line 618:

  [Symfony\Component\Console\Exception\CommandNotFoundException]
  Command "./core/scripts/run-tests.sh" is not defined.


Exception trace:
 Symfony\Component\Console\Application->find() at /var/www/html/vendor/symfony/console/Application.php:229
 Symfony\Component\Console\Application->doRun() at /var/www/html/vendor/symfony/console/Application.php:148
 Symfony\Component\Console\Application->run() at /var/www/html/vendor/consolidation/robo/src/Runner.php:198
 Robo\Runner->run() at /var/www/html/vendor/acquia/blt/src/Robo/Blt.php:236
 Acquia\Blt\Robo\Blt->run() at /var/www/html/vendor/acquia/blt/bin/blt-robo-run.php:33
 require_once() at /var/www/html/vendor/acquia/blt/bin/blt-robo.php:14
 require_once() at /var/www/html/vendor/acquia/blt/bin/blt:11

0.26s total time elapsed.
```

But calling the _same_ command directly works:

```
vagrant@local:/var/www/html/docroot$ /usr/bin/php ./core/scripts/run-tests.sh --xml /var/www/html/reports/phpunit --verbose --all --url 'http://local.html.com' --types PHPUnit-Unit User System in /var/www/html/docroot

Drupal test run
---------------

All tests will run.

Test run started:
  Friday, November 16, 2018 - 22:41

Test summary
------------

Drupal\Tests\Component\Utility\ArgumentsResolverTest          14 passes

{...}

```

Reply:
I think running with `sudo` will fix this, actually.